app/javascript/components/pages/FateNew.js

```diff
@@ -15,68 +14,127 @@ class FateNew extends Component{
         description: "",
         is_public: false,
       },
-      submitted: false
-    }
+      submitted: false,
+      urls: [],
+    };
   }
 
   handleChange = (e) => {
-        // destructuring form out of state
-        const { form } = this.state
-        form[e.target.name] = e.target.value
-        // setting state to the updated form
-        this.setState({ form: form })
-    }
-    handleSubmit = (e) => {
-        e.preventDefault()
-        this.props.createNewEvent(this.state.form)
-        this.setState({submitted: true})
-    }
+    // destructuring form out of state
```

Comment:
Not a fan of comments like these that literally describe the next line. The goal is to have your code explain itself, which, it does in this case. There's no need to comment that you're destructuring cause it's clear that's what's happening. Same with your comment about setting state. It literally is described in the code. Comments are good to make things clear when they otherwise wouldn't. You dont need to make that change now, just a comment.

Reply:
Fixed